build/docker/db2/Dockerfile

```diff
@@ -6,8 +6,7 @@
 
 FROM ibmcom/db2
 
-# Create a non-admin user for the FHIR server
-# This non-admin user will be used by fhir server to access db2
+# Create a non-admin user for the FHIR server to use to access db2
```

Comment:
There has to be at least one link / reference to this file in an md

Reply:
Not sure I follow this comment...

Reply:
sorry - I did a quick search in the docs, and we should probably make sure (since we move this file) that there are no documents that need updating

Reply:
good call.  i just found one:
```
fhir-install/README-DB2.md:docker build -t fhirserverdb2 . -f Dockerfile-db2 --squash
```
will fix

Reply:
ok, i basically ended up re-writing this readme and added it to the build dir